test/jdk/java/io/FileOutputStream/OpenNUL.java

```diff
@@ -26,7 +26,9 @@
  * @bug 8285445
  * @requires (os.family == "windows")
  * @summary Verify behavior of opening "NUL:" with ADS enabled and disabled.
+ * @run main/othervm OpenNUL
  * @run main/othervm -Djdk.io.File.enableADS OpenNUL
+ * @run main/othervm -Djdk.io.File.enableADS=FalsE OpenNUL
```

Comment:
Is there a reason for the mixed case? Just to check equalsIgnoreCase is being used?

Reply:
Yes only for equalsIgnoreCase